src/js/monkeys/mobile.js

```diff
@@ -20,6 +20,9 @@ mobile.generateHtml = function (data) {
   var $inner = $('<div />').appendTo($images)
     .addClass('landscape-images-inner');
 
+  var $letterContainer = $('#monkey')
```

Comment:
It's [overridable](https://github.com/Lostmyname/monkey/blob/master/src/js/monkey.js#L18), so not necessarily. If we wanted to make a page which demonstrates both the mobile and desktop monkeys, this would break it.

I'm pretty sure it's already saved somewhere like `data.$html`.